src/Symfony/Component/HttpClient/Tests/AmpHttpClientTest.php

```diff
@@ -24,6 +24,26 @@ protected function getHttpClient(string $testCase): HttpClientInterface
         return new AmpHttpClient(['verify_peer' => false, 'verify_host' => false, 'timeout' => 5]);
     }
 
+    /**
+     * @requires PHP 8.4
```

Comment:
Can we run this test with amphp v4 also? It'd be nice to prove it works there also (it should)

Reply:
yes should be work as expected